packages/components/src/navigation/stories/index.js

```diff
@@ -30,6 +30,12 @@ function Example() {
 	const [ activeItem, setActiveItem ] = useState( 'item-1' );
 	const [ activeMenu, setActiveMenu ] = useState( 'root' );
 
+	const [ delayedBadge, setDelayedBadge ] = useState();
+	useEffect( () => {
+		const timeout = setTimeout( () => setDelayedBadge( 2 ), 1500 );
+		return () => clearInterval( timeout );
```

Comment:
Should this be `clearTimeout` instead of `clearInterval` ?  I don't think using `clearInterval` to clear a timeout will be guaranteed to work in all cases.

Reply:
Good catch! My bad 😬 I was testing with setInterval and later on replaced it with setTimeout but forgot the clear* 😞 I'll open a PR to fix it.